src/Container/DispatchMiddlewareFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
```

Comment:
No new line before strict_type declaration please.

Reply:
@webimpress let's try reducing noise over CS by moving these discussions to the CS checks repo instead 👍

Reply:
I'm also little bit unhappy with this type of comments. Because the first comments are always related to the coding standard. After the _noise_, there are often no other comments. CS is important, but code formatting should not have the first place in this and other discussions. (imo)

Reply:
_Coding Standard Police_ 😉

Reply:
This is why we need proper CS for zf and then we are not going to have these comments at all. Sorry, I'm adding comments about everything I can see at the beginning because otherwise (it happened already couple times) it could be merged without fixing...

Reply:
@webimpress 
Sorry, my comment addressed not only to you. It was a hint to: zendframework/zend-coding-standard/pull/1

Reply:
Sorry guys, actually I was wrong... Probably we should keep the line. See my comments there:
https://github.com/zendframework/zend-expressive-aurarouter/pull/30#discussion_r164587191